src/main/java/sirius/biz/jobs/batch/file/DictionaryBasedArchiveImportJob.java

```diff
@@ -138,6 +139,9 @@ protected void importEntries() throws Exception {
         handledFiles.clear();
 
         for (ImportFile importFile : importFiles) {
+            if (!TaskContext.get().isActive()) {
```

Comment:
add same check in L155 or is this already checked by extractAllFiles?

Reply:
We return and exit the whole method, not only the for-loop.

Reply:
yes but this could remain true and turn to false in the loop below...